profile-management/Podfile

```diff
@@ -0,0 +1,6 @@
+platform :ios, '12.0'
```

Comment:
Probably OK because this is a sample, but this version should be the same [minimum as `OktaOidc`](https://github.com/okta/okta-oidc-ios/blob/master/OktaOidc.podspec#L14)